lighthouse-core/report/v2/renderer/category-renderer.js

```diff
@@ -116,6 +116,109 @@ class CategoryRenderer {
   }
 
   /**
+   * @param {!ReportRenderer.AuditJSON} audit
+   * @param {number} scale
+   * @return {!Element}
+   */
+  _renderPerfHintAudit(audit, scale) {
+    const extendedInfo = /** @type {!CategoryRenderer.PerfHintExtendedInfo}
+        */ (audit.result.extendedInfo);
+    if (!extendedInfo.value.wastedMs) return this._dom.createElement('span');
+
+    const element = this._dom.createElement('details', [
+      'lh-perf-hint',
+      `lh-perf-hint--${Util.calculateRating(audit.score)}`,
+      'lh-expandable-details',
+    ].join(' '));
+    const summary = this._dom.createElement('summary', 'lh-perf-hint__summary ' +
+        'lh-expandable-details__summary');
+    element.appendChild(summary);
+
+    const titleEl = this._dom.createElement('div', 'lh-perf-hint__title');
+    titleEl.textContent = audit.result.description;
+    summary.appendChild(titleEl);
+
+    const sparklineContainerEl = this._dom.createElement('div', 'lh-perf-hint__sparkline');
+    const sparklineEl = this._dom.createElement('div', 'lh-sparkline');
+    const sparklineBarEl = this._dom.createElement('div', 'lh-sparkline__bar');
+    sparklineBarEl.style.width = extendedInfo.value.wastedMs / scale * 100 + '%';
+    sparklineEl.appendChild(sparklineBarEl);
```

Comment:
fwiw you do have `this._dom.createChildOf` if you want to nuke these append lines.